include/wx/math.h

```diff
@@ -181,4 +181,24 @@ inline double wxRadToDeg(double rad) { return (rad * 180.0) / M_PI; }
 /* Compute the greatest common divisor of two positive integers */
 WXDLLIMPEXP_BASE unsigned int wxGCD(unsigned int u, unsigned int v);
 
+
+
+/* Count trailing zeros. Use optimised builtin where available. */
+/* Count trailing zeros. Use optimised builtin where available. */
+inline unsigned int wxCtz(unsigned x)
```

Comment:
This should be inside `#ifdef __cplusplus` section.

Reply:
I'd still put it there because it's not (and shouldn't be) `extern "C"`, so it's not callable from C anyhow.